math/src/field/f64/mod.rs

```diff
@@ -3,7 +3,9 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
-//! An implementation of a 64-bit STARK-friendly prime field with modulus $2^{64} - 2^{32} + 1$.
+//! An implementation of a 64-bit STARK-friendly prime field with modulus $2^{64} - 2^{32} + 1$
+//! using Montgomery representation.
+//! Our implementation follows https://eprint.iacr.org/2022/274.pdf
```

Comment:
Maybe mention in this comment that most (all?) functions here are constant-time?